README.md

```diff
@@ -198,3 +198,61 @@ bash script/local/deploy_query_local.sh
 ```
 
 This will print out verbose logs of the deployment, including the addresses of **multiple** deployed contracts (SNARK verifier, historical SNARK verifier, query MMR SNARK verifier, `AxiomV1`, `AxiomV1Query`, and `AxiomProxy`).
+
+### Code Analysis for contributors
+
+To generate documents to better understand and analyze the code we can run
+
+```bash
+bash script/local/code_analysis.sh
+```
+
+This generates the following files (links and samples below)
+
+#### [axiom-v1-inheritance-contracts-inheritance-graph.png](./docs/axiom-v1-contracts-inheritance-graph.png)
+
+Useful for undertanding the axiom-v1-contracts their functions and relationships
+
+![axiom-v1-inheritance-contracts-inheritance-graph.png](./docs/axiom-v1-contracts-inheritance-graph.png)
+
+#### [axiom-v1-contracts-coverage.log](./docs/axiom-v1-contracts-coverage.log)
+
+Useful for reviewing test coverage for axiom contracts
+
+```text
+Compiling 90 files with 0.8.19
+Solc 0.8.19 finished in 6.63s
+Compiler run [32msuccessful![0m
```

Comment:
is the [32m [0m intentional?

Reply:
It is a carryover from piping the output from the slither command
`slither . --exclude-informational --exclude-low &> ./docs/axiom-v1-contracts.log`
If we want to remove it, I can look at removing the lines by editing the file after the command.